core/ir/disassemble_shared.c

```diff
@@ -809,6 +817,48 @@ print_extra_bytes_to_file(file_t outfile, byte *pc, byte *next_pc, int extra_sz,
     os_write(outfile, buf, sofar);
 }
 
+/* DR_ISA_REGDEPS instruction encodings can be at most 16 bytes,
+ * hence we can have at most 2 lines.
+ */
+#define REGDEPS_BYTES_PER_LINE 8
+
+int
+print_regdeps_encoding_bytes_to_buffer(char *buf, size_t bufsz,
+                                       size_t *sofar DR_PARAM_INOUT, byte *pc,
+                                       byte *next_pc)
+{
+    int sz = (int)(next_pc - pc);
+    int i, extra_sz;
+    if (sz > REGDEPS_BYTES_PER_LINE) {
+        extra_sz = sz - REGDEPS_BYTES_PER_LINE;
+        sz = REGDEPS_BYTES_PER_LINE;
+    } else
+        extra_sz = 0;
+    for (i = 0; i < sz; i++)
+        print_to_buffer(buf, bufsz, sofar, " %02x", *(pc + i));
```

Comment:
Do we want 1-byte separate bytes?  Every instruction has a 4-byte header: seems more natural to print a 4-byte word for that, right?

Reply:
Yes, 4 byte words makes more sense. Done.